### ISSUE TITLE
AI is too confident.

### ISSUE THREAD

**Issue by xuuyz** [flurb18/AgentOoba#4]
### Human:
Objectives:
Objective 1 is: Find out what the Stable Diffusion AI is, and give links on how to access it
Objective 1 is the current task we are working on.

Instructions:
Assess whether a large language model is capable of completing Objective 1.
Remember that large language models are not able to perform any physical tasks, interact with the world, or use the internet.
If a large language model cannot complete Objective 1, respond with the word 'No'.
If a large language model could help partially complete Objective 1 but not entirely, respond with the word 'No'.
If you are unsure or need clarification, respond with the word 'No'.
Otherwise, if you are certain that a large language model could complete Objective 1 with a single output, respond with the word 'Yes'.
Your response should only be either the word 'No' or the word 'Yes', depending on the criteria above.

### Assistant:

----------------------OUTPUT-----------------------
 Yes

Output generated in 8.13 seconds (9.22 tokens/s, 75 tokens, context 102, seed 1572502792)
-----------------------INPUT-----------------------
### Human:
Objectives:
Objective 1 is: Find out what the Stable Diffusion AI is, and give links on how to access it
Objective 1 is the current task we are working on.

Instructions:
Achieve Objective 1, and respond with the text that Objective 1 requires as output.
Do not respond with anything but the output that Objective 1 requires.

### Assistant:

----------------------OUTPUT-----------------------
 The Stable Diffusion AI is a language model trained by researchers from Large Model Systems Organization (LMSYS). It can generate human-like responses to prompts and has been made available for use through an API. You can find more information about accessing the Stable Diffusion AI and using its API on the LMSYS website.

**Comment by flurb18** [comment]
Agreed. Customizing the prompts to the right degree is hard. The default prompts are pretty good, but if you've got a better one now you can customize it yourself in the UI and export them as a file. I'd love to see different prompt templates and how they perform.
